### PR TITLE
tweak-deps

### PR DESCRIPTION
**Pull request by axrs**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,15 @@
...
+
+* Updated Dependencies:
+  * `bindle` submodule to latest `master`
+  * `io.jesi/parent` to `4.12.0`
+  * `io.jesi/backpack` to `7.1.0`
+  * `leiningen/leiningen` to `2.9.6`
+  * `org.clojure/tools.namespace` to `1.1.0`
+  * `pjstadig/humane-test-output` to `0.11.0`
+* Removed Managed Dependencies:
+  * `com.google.guava/guava` to `4.11.0`
```
Comment (axrs) [inline CHANGELOG.md:11]: fix and update others

File: test/io/jesi/customs/strict_test.cljc [hunk 2]
```diff
@@ -71,21 +71,36 @@
...
-        (macroexpand '(io.jesi.customs.strict/is true))))
+                               (cljs.test/report {:type       :error
+                                                  :expected   'true
+                                                  :actual     t#
+                                                  :message    nil
+                                                  :file       nil
+                                                  :end-column nil
+                                                  :column     nil
+                                                  :line       nil
+                                                  :end-line   nil}))))
```
Comment (axrs) [inline test/io/jesi/customs/strict_test.cljc:102]: some extra attrs now exist

File: test/io/jesi/customs/strict_test.cljc [hunk 3]
```diff
@@ -232,7 +262,7 @@
 
     (testing "fails if empty"
       (let [report (atom nil)]
-        (with-redefs [test/do-report (partial reset! report)]
+        (with-redefs [test/report (partial reset! report)]
```
Comment (axrs) [inline test/io/jesi/customs/strict_test.cljc:265]: `do-report` is no longer a thing

File: project.clj [hunk 4]
```diff
@@ -7,20 +7,19 @@
...
+                 [org.clojure/tools.namespace "1.1.0"]
                  ;CLJS
                  [org.clojure/clojurescript :scope "provided"]
-                 [pjstadig/humane-test-output "0.10.0"]]
-  :profiles {:dev [:parent/dev {:dependencies [[lein-parent "0.3.7"]
+                 [pjstadig/humane-test-output "0.11.0"]]
+  :profiles {:dev [:parent/dev {:dependencies [[lein-parent/lein-parent "0.3.8"]
                                                [org.slf4j/slf4j-simple "1.7.30"]
-                                               [thheller/shadow-cljs]]}]})
+                                               [thheller/shadow-cljs :scope "provided"]]}]})
```
Comment (AndreTheHunter) [inline project.clj]: I don't think we wan "provided" for a dev dependency
```suggestion
                                               [thheller/shadow-cljs]]}]})
```

**Review by AndreTheHunter — CHANGES_REQUESTED** [review]

File: project.clj (hunk 4 above)
Comment (axrs) [reply project.clj]: Good point!

**Review by AndreTheHunter — APPROVED** [review]

**Pull request merged**